flink-filesystems/flink-fs-hadoop-shaded/src/main/resources/META-INF/NOTICE

```diff
@@ -33,7 +48,7 @@ See bundled license files for details.
 
 - org.codehaus.woodstox:stax2-api:3.1.4 (https://github.com/FasterXML/stax2-api/tree/stax2-api-3.1.4)
 
-This project bundles org.apache.hadoop:*:3.1.0 from which it inherits the following notices:
+This project bundles org.apache.hadoop:*:3.3.1 from which it inherits the following notices:
 
 The Apache Hadoop project contains subcomponents with separate copyright
```

Comment:
did you check if the notices changed?

Reply:
@AHeise How do i check if the notices have changed. Any pointers are appreciated. Thank you very much.

Reply:
I think you did a good job already. We also have some tooling on each CI run that does some checks.